docs/tutorials/htcondor.md

```diff
@@ -0,0 +1,233 @@
+> blueprint variables. The `--vars` argument supports comma-separated list of
+> name=value variables to override blueprint variables. This feature only
+> supports variables of string type.
+
+This will create a deployment directory named `htcondor-001/`, which
+contains the terraform needed to deploy your cluster.
+
+## Deploy the Cluster
+
+Use the following commands to run terraform and deploy your cluster.
+
+```bash
+terraform -chdir=htcondor-001/htcondor init
+terraform -chdir=htcondor-001/htcondor validate
+terraform -chdir=htcondor-001/htcondor apply -auto-approve
+```
+
+If you receive any errors during `apply`, you may re-run it to resolve them.
+The deployment will take about 3 minutes. There should be regular status updates
+in the terminal. If the `apply` is successful, a message similar to the
+following will be displayed:
+
+<!-- Note: Bash blocks give "copy to cloud shell" option.  -->
+<!-- "shell" or "text" is used in places where command should not be run in cloud shell. -->
+
+```shell
+Apply complete! Resources: xx added, 0 changed, 0 destroyed.
+```
+
+## Wait for the pool to be ready
+
+Once terraform has finished, you may SSH to the HTCondor Access Point:
+
+```bash
+gcloud compute ssh access-point-0 --tunnel-through-iap --project <walkthrough-project-id/> --zone us-central1-c
+```
+
+Alternatively, you may browse to the `access-point-0` VM and click on "SSH" in
+the Cloud Console at this address:
+
+```text
+https://console.cloud.google.com/compute?project=<walkthrough-project-id/>
+```
+
+Once you have command line access to the machine, you must wait for HTCondor and
+Docker to complete installation. This will take approximately 5 minutes from
+when `terraform apply` finished. When it is complete, a message will be printed
+to the screen:
+
+```text
+******* HTCondor system configuration complete ********
+```
+
+You should also verify that the pool is operational by executing
+
+```bash
+condor_status -schedd -autoformat Name
+```
+
+and observing output similar to
+
+```text
+access-point-0.us-central1-c.c.<walkthrough-project-id/>.internal
+```
+
+## Submit an example job
+
+The following commands will copy this job into your home directory and submit it
+to the HTCondor pool.
+
+```text
+universe       = docker
+docker_image   = hello-world
+output         = out.$(Cluster)-$(Process)
+error          = err.$(Cluster)-$(Process)
+log            = log.$(Cluster)-$(Process)
+request_cpus   = 1
+request_memory = 100MB
+queue
+```
+
+```bash
+cp /var/tmp/helloworld.sub .
+condor_submit helloworld.sub
+```
+
+The output should resemble
+
+```text
+Submitting job(s).
+1 job(s) submitted to cluster 1.
+```
+
+Run `condor_watch_q` to watch your jobs as they transition from `IDLE` to `RUN`
+to `DONE`. This will take several minutes are the pool autoscales to serve your
+job.
+
+```bash
+condor_watch_q
+```
+
+The output should resemble
+
+```text
+BATCH   IDLE  RUN  DONE  TOTAL  JOB_IDS
+ID: 1     1    -     -      1   1.0
```

Comment:
nit: The job took longer to run than I would have expected without comment on the length. It might be ince to say, "normally it takes about xx minutes for the job to run".

Reply:
There is some text above this section but I've emphasized it and added a bit more after.